code/modules/admin/admin.dm

```diff
@@ -724,13 +725,13 @@ GLOBAL_VAR_INIT(nologevent, 0)
 	if(!check_rights(R_SERVER))
 		return
 
-	GLOB.guests_allowed = !( GLOB.guests_allowed )
-	if(!( GLOB.guests_allowed ))
+	GLOB.configuration.general.guest_ban = !(GLOB.configuration?.general.guest_ban)
```

Comment:
Is the usage of `?.` here and below needed?

Reply:
This is indeed a weird usage of `?.`, because the left side assignment would fail.

Reply:
I dont know why I `?.`'d it here. I assumed it was a carryover, but in this case, config can never be null here, as it is loaded in world/New(), and you cannot call this verb without connecting.